spec/convection/model/cloudfile_spec.rb

```diff
@@ -3,11 +3,32 @@
 
 module Convection::Model
   describe Cloudfile do
+    let(:template) do
+      Convection.template do
+        description 'EC2 VPC Test Template'
+
+        ec2_vpc 'TargetVPC' do
+          network '10.0.0.0'
+          subnet_length 24
+          enable_dns
+        end
+      end
+    end
+
     describe 'stack' do
       subject do
-        klass = Class.new
-        klass.extend(Convection::DSL::Cloudfile)
-        klass
+        # initialize does too much to call that directly
+        # setup only what we need for testing... :/
```

Comment:
Out of scope for this PR but — Would be nice to move a lot of the stuff it doesn't into a method we can stub for unit testing. Doesn't provide any functional benefit it would be more just developer friendliness.